### ISSUE TITLE
calculate the loss in fine-tuning stage 

### ISSUE THREAD

**Issue by backtime92** [clovaai/CRAFT-pytorch#33]
how to calculate the loss in fine-tuning stage? when I process weakly-supervised training stage that I calculate the loss of each image in the batch and then add them divide batch size, is that right?

**Comment by YoungminBaek** [comment]
@backtime92 Yes, that's right.
We did the same thing as you said; calculate loss, add them, and divide by batch size.
